2023-risks/M8: Insufficient Cryptography

```diff
@@ -0,0 +1,106 @@
+---
+
+layout: col-sidebar
+title: "M8: Insufficient Cryptography"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Individuals who possess physical proximity to data that has been inadequately encrypted or mobile malware that operates on behalf of an adversary can compromise the security of the system.
+
+# Attack vectors
+
+**Exploitability Average**
+
+Any API call that is accessible to potential attackers can be exploited as an attack vector. An attack vector refers to a specific method or pathway through which an attacker can gain unauthorized access, manipulate data, or compromise the security of the mobile application.
```

Comment:
I would also mention persistent storage on the device as attack target. At least you also list it for the threat agents.